src/class/Battle.js

```diff
@@ -0,0 +1,10 @@
+export default class Battle {
+
+    constructor( monsters, fighters){
+       
```

Comment:
this.monsters = monsters 
this.fighters = fighters

Reply:
je vous remercie